src/database/seeders/20220302143415-demo-user.js

```diff
@@ -1,30 +1,24 @@
-'use strict';
-
 module.exports = {
-    async up(queryInterface, Sequelize) {
-        return queryInterface.bulkInsert('Users', [{
-                firstName: 'John',
-                lastName: 'Doe',
-                email: 'example@example.com',
-                createdAt: new Date(),
-                updatedAt: new Date(),
-            },
-            {
-                firstName: 'andela',
-                lastName: 'kigali',
-                email: 'andela@andela.com',
-                createdAt: new Date(),
-                updatedAt: new Date(),
-            },
-        ]);
-    },
+  up(queryInterface, Sequelize) {
+    queryInterface.bulkInsert('Users', [
+      {
+        firstName: 'John',
+        lastName: 'Doe',
+        email: 'example@example.com',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        firstName: 'andela',
+        lastName: 'kigali',
+        email: 'andela@andela.com',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+    ]);
+  },
```

Comment:
Why exactly are you seeding this user?

Reply:
Updates the seeders to have one user that maybe seeded in case a user is needed in the database and the create endpoint is not yet created or has an issue.

Reply:
Updates the seeders to have one user that maybe seeded in case a user is needed in the database and the create endpoint is not yet created or has an issue.